src/Backend/Modules/Partners/Actions/Edit.php

```diff
@@ -22,7 +22,9 @@ public function execute()
         }
 
         $editWidget = new UpdateWidget($widget);
-        $form = $this->createForm(WidgetType::class, $editWidget);
+
+        $theme = $this->get('fork.settings')->get('Core', 'theme', 'core');
+        $form = $this->createForm(new WidgetType($theme), $editWidget);
```

Comment:
dit is deprecated en mag dus niet meer, als je het zo wilt doen omdat je met fork settings niet makkelijk als service kan werken moet je het doorgeven als form option